core/dbt/config/errors.py

```diff
@@ -0,0 +1,8 @@
+from contextlib import contextmanager
```

Comment:
what's up with this file?

Reply:
This file is what happens when you `git add dbt/core` and have dead code lying around...